### PR TITLE
Add python functions documentation

### PR DESCRIPTION
**Pull request by koustuvsinha**
Converting python tutorial docs from https://github.com/alayek/python3notes from ipynb to markdown, by preserving formatting. Continuing work from [this PR](https://github.com/alayek/python3notes/pull/1)

### CODE REVIEW

File: python-functions.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+>>> def greater_than_1(n):
+...     return n > 1
+...
+>>> print(greater_than_1(1))
+False
+>>> print(greater_than_1(2))
+True
+```
+
+If no expression list is specified, None is returned:
```
Comment (alayek) [inline python-functions.md]: `None` is returned.

**Comment by alayek** [comment]
Nice job @koustuvsinha @evaristoc !

We would expand on the links and add those sections later. But for now, I am okay with it.

@raisedadead @Rafase282 please review.

**Comment by Rafase282** [comment]
Please update the title and links as I had previously state in the other PR. @koustuvsinha also great work so far.

**Comment by QuincyLarson** [comment]
@koustuvsinha Technically these commits should be squashed. I'll be honest - I want to get this live ASAP for the big announcement.

**Comment by koustuvsinha** [comment]
@QuincyLarson i will keep that in mind :)

**Pull request merged**